mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po

```diff
@@ -1956,3 +1956,15 @@ msgstr "@*!^: 0H... @ n3wc0m3r! H0vv p1d y0v g3t he7e? <NONE> h3llen\"T $33N %^&
 
 msgid "cannot_use_item_monster"
 msgstr "This item cannot be used on this monster."
+
+msgid "taba_greeting1"
+msgstr "Greeter Jaime: Hey there ${{name}}! Heard you got a Tuxemon, congrats! If you wanna grow your party, you'll need Tuxeballs."
```

Comment:
How does the greeter know your name? Maybe something more like, "Congratulations on being the one-thousandth customer!"

Reply:
I think I had it in my head that they would know you because this is the shop your family would use, so he would have seen you on the regular.